components/core/JsInterop/interop.ts

```diff
@@ -169,61 +179,216 @@ export function delElementFrom(delElement, elementSelector) {
+        } else {
+            elements = selector;
+        }
+        if (elements) {
+            for (var i = 0; i < elements.length; i++) {
+                elements[i].parentNode.removeChild(elements[i]);
+            }
+        }
+    }
+}
+
+/**
+ * ���selector��Ԫ��
+ * @param selector string(cssѡ����) | Element | NodeList
+ */
+export function emptyElement(selector: string | Element | NodeList) {
+    if (selector instanceof Element) {
+        selector.innerHTML = "";
+    }
+    let elements: NodeList;
+
+    if (typeof selector === "string") {
+        elements = getDoms(selector);
+    } else if (selector instanceof NodeList){
+        elements = selector;
+    }
+    if (elements) {
+        for (var i = 0; i < elements.length; i++) {
+            if (elements[i] instanceof Element) {
+                (<Element>elements[i]).innerHTML = "";
+            }
+        }
+    }
+}
+
+/**
+ * ��ȡselector��class�б�
+ * @param selector string(cssѡ����) | Element
+ * @return class name array
+ */
+export function getClass(selector: string | Element) {
+    let element: Element;
+    if (typeof selector === "string") {
+        element = getDom(selector);
+    } else {
+        element = selector;
+    }
+    let classNameArr = element.getAttribute("class").split(" ").map((item) => {
         return item.trim();
-    }).filter(item => {
-        return item !== "ant-notification-fade-enter" && item !== "ant-notification-fade-enter-active";
-    }).join(" ");
+    });
+    return classNameArr;
+}
+
+/**
+ * ��selector�е�class���Ƴ�className
+ * @param selector string(cssѡ����) | Element
+ * @param className string(class name) | Array<string>(class name array)
+ */
+export function removeClass(selector: string | Element,
+    className: string | Array<string>) {
+    let element: Element;
+    if (typeof selector === "string") {
+        element = getDom(selector);
+    } else {
+        element = selector;
+    }
+
+    let classNameArr = getClass(element);
+    if (typeof className === "string") {
+        classNameArr = classNameArr.filter((item: string) => {
+            return item !== className;
+        });
+    } else {
+        classNameArr = classNameArr.filter((item: string) => {
+            return !className.includes(item);
+        });
+    }
+    let newClassName = classNameArr.join(" ");
     element.setAttribute("class", newClassName);
+    return element;
+}
+
+/**
+ * ��js��ע��C#���ʵ������
+ * @param instance
+ * @param instanceName
+ */
+export function initCsInstance(instance: any, instanceName: string) {
+    let csInstance = (<any>window).antBlazor.interop.csInstance;
+    if (!csInstance) {
+        csInstance = {};
+        (<any>window).antBlazor.interop.csInstance = csInstance;
+    }
+    csInstance[instanceName] = instance;
+}
+
+/**
+ * ɾ��js��ע���C#���ʵ������
+ * @param instanceName
+ */
+export function delCsInstance(instanceName: string) {
+    let csInstance = (<any>window).antBlazor.interop.csInstance;
```

Comment:
In TS, you can declare `notificationService` and `csInstance ` as the export module.
Don't define it directly on `window`.

Reply:
OK, I'll change it.